app/models/concerns/reviewable.rb

```diff
@@ -0,0 +1,21 @@
+# frozen_string_literal: true
+
+module Reviewable
+  extend ActiveSupport::Concern
+
+  included do
+    belongs_to :assessment
+
+    validates :reviewed_at, presence: true, unless: -> { passed.nil? }
+
+    def reviewed!(passed)
+      update!(passed:, reviewed_at: Time.zone.now)
+    end
+
+    def review_state
```

Comment:
I wonder if this could simply be called `status` since I think we can use it as the status in all situations where we want to render the status tag next to this reviewable item.